README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Build Status](https://circleci.com/gh/micrometer-metrics/micrometer.svg?style=shield)](https://circleci.com/gh/micrometer-metrics/micrometer)
 [![Apache 2.0](https://img.shields.io/github/license/micrometer-metrics/micrometer.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+[![Maven Central](https://img.shields.io/maven-central/v/io.micrometer/micrometer-core.svg)](https://mvnrepository.com/artifact/io.micrometer/micrometer-core)
```

Comment:
Just a thought: why don't we link to the official Maven central site? That is: https://search.maven.org/artifact/io.micrometer/micrometer-core
I used to prefer mvnrepository.com's UI to the official Maven central site, but since they've revamped the UI, I think it is pretty good. Is there any reason to prefer the unofficial site for this?

Reply:
@shakuzen I didn't give much thought on this. The link has been picked just as it's been usually searched on top in Google. I'm okay with either way. Do you prefer the official one for this? If so, I'll update that way. Just let me know.

Reply:
I think it is fine for now. We can reconsider late if it comes up again.